core/templates/domain/story/story-update.service.ts

```diff
@@ -65,18 +73,20 @@ export class StoryUpdateService {
       apply: StoryUpdateApply, reverse: StoryUpdateReverse): void {
     let changeDict = cloneDeep(params) as BackendChangeObject;
     changeDict.cmd = command;
-    let changeObj = new Change(changeDict, apply, reverse);
+    let changeObj = new Change(changeDict, apply as Apply, reverse as Reverse);
     try {
       this._undoRedoService.applyChange(changeObj, story);
-    } catch (err) {
-      this._alertsService.addWarning(err.message);
+    } catch (err: unknown) {
```

Comment:
Please add a comment why is it `unknown` here.

Reply:
Done.